chain/jsonrpc/src/lib.rs

```diff
@@ -399,11 +399,12 @@ impl JsonRpcHandler {
                         ))))
                     }
                 };
-                RpcQueryRequest { block_id: None, request }
+                RpcQueryRequest { block_id: None, request, finality: Finality::NFG }
```

Comment:
This looks like a breaking change behavior to the old API, which used to query the latest block. Let’s use Finality::None